CRITICAL - 2021-12-16 00:05:01 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\coralistest\app\Views\Pages\v_login.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\coralistest\system\View\View.php(214): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_login', Array, true)
#4 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(27): view('pages/v_login', Array)
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->index()
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-12-16 00:22:41 --> The original file is not a valid file.
#0 C:\xampp\htdocs\coralistest\system\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
#1 C:\xampp\htdocs\coralistest\app\Controllers\Register.php(34): CodeIgniter\HTTP\Files\UploadedFile->move('./img/photoprof...', '1639635761_10ef...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-12-16 00:24:45 --> Undefined property: App\Controllers\Register::$hasMoved
#0 C:\xampp\htdocs\coralistest\app\Controllers\Register.php(34): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 34, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 00:25:25 --> Cannot access protected property CodeIgniter\HTTP\Files\UploadedFile::$hasMoved
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 00:25:30 --> Cannot access protected property CodeIgniter\HTTP\Files\UploadedFile::$hasMoved
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 00:28:35 --> Undefined property: CodeIgniter\HTTP\Files\UploadedFile::$isValid
#0 C:\xampp\htdocs\coralistest\app\Controllers\Register.php(34): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 34, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 00:29:15 --> The original file is not a valid file.
#0 C:\xampp\htdocs\coralistest\system\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
#1 C:\xampp\htdocs\coralistest\app\Controllers\Register.php(38): CodeIgniter\HTTP\Files\UploadedFile->move('./img/photoprof...', '1639636155_c2ac...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->store()
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-12-16 00:30:06 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\coralistest\app\Views\Pages\v_register.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\coralistest\system\View\View.php(214): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_registe...', Array, true)
#4 C:\xampp\htdocs\coralistest\app\Controllers\Register.php(22): view('pages/v_registe...', Array)
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Register->index()
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#7 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-12-16 02:46:19 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::get_where()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('riskidwipatrio1...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:46:37 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('riskidwipatrio1...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:48:59 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('riskidwipatrio1...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:49:00 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('riskidwipatrio1...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:49:55 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:53:29 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:53:47 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:53:48 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:53:49 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:54:09 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::getWhere()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(128): App\Models\UserModel->getUserInfo('11')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 02:56:30 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::insert_string()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): App\Models\UserModel->insertToken('10')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:23:27 --> Trying to get property 'user_id' of non-object
#0 C:\xampp\htdocs\coralistest\app\Models\UserModel.php(94): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 94, Array)
#1 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(132): App\Models\UserModel->isTokenValid('74384b5368299a7...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-12-16 03:31:58 --> Undefined property: App\Controllers\Login::$m_account
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(101): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 101, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->forgotPassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:32:33 --> Call to undefined method App\Controllers\Login::base64url_encode()
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->forgotPassword()
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
ERROR - 2021-12-16 03:33:07 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 03:33:35 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 03:34:07 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
CRITICAL - 2021-12-16 03:38:54 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(135): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 135, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:39:23 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(135): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 135, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:39:35 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(135): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 135, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:40:17 --> Undefined property: App\Controllers\Login::$URI
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(135): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 135, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:40:35 --> Undefined property: App\Controllers\Login::$URI
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(136): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 136, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:41:25 --> Undefined property: App\Controllers\Login::$URI
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(136): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 136, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:41:25 --> Undefined property: App\Controllers\Login::$URI
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(136): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 136, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:42:40 --> Call to undefined method CodeIgniter\HTTP\URI::segment()
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 03:43:27 --> Call to undefined method App\Controllers\Login::getVar()
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 03:43:40 --> Call to undefined method App\Controllers\Login::get()
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 03:44:01 --> Call to undefined method App\Controllers\Login::get()
#0 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ZDRmYjM0ZjdiYjQ...')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-12-16 03:47:51 --> syntax error, unexpected '}'
#0 C:\xampp\htdocs\coralistest\system\Router\Router.php(176): CodeIgniter\Router\Router->autoRoute('login/a')
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(718): CodeIgniter\Router\Router->handle('login/a')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(359): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:49:09 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(131): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:50:49 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(131): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:51:15 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(131): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:51:24 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(131): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 03:52:09 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 130, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 05:26:42 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 130, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 05:26:46 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 130, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a('a')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 05:26:48 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 130, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a('a')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 05:41:54 --> Undefined property: App\Controllers\Login::$input
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(130): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 130, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->a('a')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 05:59:48 --> Undefined property: App\Controllers\Login::$uri
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(131): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 06:09:26 --> Undefined property: App\Controllers\Login::$security
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(133): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 133, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test('2')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 06:09:43 --> Undefined property: CodeIgniter\HTTP\IncomingRequest::$security
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(133): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 133, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->test('2')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-12-16 06:11:32 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
CRITICAL - 2021-12-16 06:21:48 --> Undefined property: App\Controllers\Login::$form_validation
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(149): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 06:47:27 --> Invalid file: /v_forgotpassword.php
#0 C:\xampp\htdocs\coralistest\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('/v_forgotpasswo...')
#1 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('/v_forgotpasswo...', Array, true)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(150): view('/v_forgotpasswo...', Array)
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 06:50:28 --> Invalid file: v_resetpassword.php
#0 C:\xampp\htdocs\coralistest\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_resetpassword...')
#1 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('v_resetpassword', Array, true)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(149): view('v_resetpassword', Array)
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:02:15 --> Invalid file: v_resetpassword.php
#0 C:\xampp\htdocs\coralistest\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_resetpassword...')
#1 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('v_resetpassword', Array, true)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(149): view('v_resetpassword', Array)
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:03:03 --> Invalid file: v_resetpassword.php
#0 C:\xampp\htdocs\coralistest\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_resetpassword...')
#1 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('v_resetpassword', Array, true)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(149): view('v_resetpassword', Array)
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:03:56 --> Invalid file: v_resetpassword.php
#0 C:\xampp\htdocs\coralistest\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_resetpassword...')
#1 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('v_resetpassword', Array, true)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(149): view('v_resetpassword', Array)
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:07:23 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\coralistest\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\coralistest\system\Common.php(840): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(138): redirect('http://localhos...', 'refresh')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:07:43 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\coralistest\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\coralistest\system\Common.php(840): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(138): redirect('http://localhos...', 'refresh')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:08:19 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\coralistest\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\coralistest\system\Common.php(840): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(138): redirect('http://localhos...', 'refresh')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:26:38 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\coralistest\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\coralistest\system\Common.php(840): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(138): redirect('http://localhos...', 'refresh')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-12-16 07:30:38 --> Undefined variable: data
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(166): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 166, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 07:30:40 --> Undefined variable: data
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(166): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 166, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword('ODcyNTYwYjBhMzg...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 07:31:54 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(160): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 07:34:01 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(158): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 158, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 07:34:30 --> Undefined variable: data
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(167): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 167, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 07:34:39 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\coralistest\system\View\View.php(214): include()
#1 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_resetpa...', Array, true)
#3 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(167): view('pages/v_resetpa...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-12-16 07:35:36 --> Array to string conversion
#0 C:\xampp\htdocs\coralistest\app\Views\Pages\v_resetpassword.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\coralistest\system\View\View.php(214): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_resetpa...', Array, true)
#4 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(169): view('pages/v_resetpa...')
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-12-16 07:37:12 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\coralistest\system\View\View.php(214): include()
#1 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_resetpa...', Array, true)
#3 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(167): view('pages/v_resetpa...')
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-12-16 08:04:02 --> Object of class CodeIgniter\Validation\Validation could not be converted to int
#0 C:\xampp\htdocs\coralistest\app\Views\Pages\v_resetpassword.php(16): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Object of class...', 'C:\\xampp\\htdocs...', 16, Array)
#1 C:\xampp\htdocs\coralistest\system\View\View.php(214): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_resetpa...', Array, true)
#4 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(167): view('pages/v_resetpa...')
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-12-16 08:18:31 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(143): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 143, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 08:19:01 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(143): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 143, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 08:20:43 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(143): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 143, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 08:20:52 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(143): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 143, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 08:21:11 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Login.php(143): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 143, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Login->resetpassword()
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-12-16 08:27:34 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 08:31:08 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
CRITICAL - 2021-12-16 08:32:06 --> Trying to access array offset on value of type bool
#0 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(45): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 45, Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('OTI4NmNkN2U0N2U...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-12-16 08:32:32 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 08:33:38 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 08:36:16 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
CRITICAL - 2021-12-16 08:54:26 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::where()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(69): App\Models\UserModel->updatePassword(Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('ODJhY2IwYzE0OWU...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-12-16 08:56:43 --> Undefined variable: token
#0 C:\xampp\htdocs\coralistest\app\Views\Pages\v_resetpassword.php(47): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 47, Array)
#1 C:\xampp\htdocs\coralistest\system\View\View.php(214): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\coralistest\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\coralistest\system\Common.php(1097): CodeIgniter\View\View->render('pages/v_resetpa...', Array, true)
#4 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(81): view('pages/v_resetpa...', Array)
#5 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('ODJhY2IwYzE0OWU...')
#6 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#7 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2021-12-16 09:15:24 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 09:17:15 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 09:20:16 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
CRITICAL - 2021-12-16 09:21:58 --> Undefined property: CodeIgniter\Database\MySQLi\Builder::$get
#0 C:\xampp\htdocs\coralistest\app\Models\UserModel.php(104): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 104, Array)
#1 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(69): App\Models\UserModel->updatePassword(Array)
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('ZDljMmIxNTQzYjM...')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-12-16 09:22:15 --> Undefined property: CodeIgniter\Database\MySQLi\Result::$affected_rows
#0 C:\xampp\htdocs\coralistest\app\Models\UserModel.php(104): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 104, Array)
#1 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(69): App\Models\UserModel->updatePassword(Array)
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('ZDljMmIxNTQzYjM...')
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#4 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-12-16 09:22:29 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::getaffected_rows()
#0 C:\xampp\htdocs\coralistest\app\Controllers\Resetpassword.php(69): App\Models\UserModel->updatePassword(Array)
#1 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(824): App\Controllers\Resetpassword->reset('ZDljMmIxNTQzYjM...')
#2 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Resetpassword))
#3 C:\xampp\htdocs\coralistest\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\coralistest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\coralistest\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-12-16 09:35:28 --> Email: sendWithMail throwed mail(): SMTP server response: 530 5.7.0 Must issue a STARTTLS command first. w16sm5906761pgh.81 - gsmtp
ERROR - 2021-12-16 09:48:39 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
ERROR - 2021-12-16 10:12:01 --> Email: sendWithMail throwed mail(): Failed to connect to mailserver at &quot;smtp.gmail.com&quot; port 587, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
